datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/cliUtils/DataframeOptions.scala

```diff
@@ -30,20 +32,51 @@ case class DataframeOptions(format: String, options: Map[String, String], path:
     if (format == "jdbc") dfReader.load()
     else dfReader.load(path)
 
-  private def save[T](dfWriter: DataFrameWriter[T], extraPath: String = ""): Unit =
+  private def save[T](dfWriter: DataFrameWriter[T], endPath: String = ""): Unit =
     if (format == "jdbc") dfWriter.save()
-    else dfWriter.save(s"$path$extraPath")
+    else dfWriter.save(endPath)
+
+  def getUniqueFilePath(extraPath: String, conf: Configuration): String = {
```

Comment:
Also naming. Wouldn't 'getUniquePath` or `generateUniquePath` be enough?

Reply:
Probably would. I think it is fine

Reply:
OK, your repo 😉